### DISCUSSION TITLE
Divide accessibility questions in Level of difficulty groups

### DISCUSSION THREAD

**Discussion by YuriDevAT** [freeCodeCamp/Developer_Quiz_Site#898] [Ideas]
## Description
Going through the list of a11y questions, I think these are mostly intermediate to advanced questions. As someone, who has no idea about accessibility it is a tough entry into this very important topic.

## Idea
Divide a11y questions into three groups: beginner, intermediate, advanced. As my experience in this field taught me (working as an Accessibility Specialist, being certified with CPACC), points as an additional entry could be added to the question (beneath the Link resources e.g.).

### Beginner
Basic (what is - ) questions like, what is accessibility, what are landmarks, layout structure, min color contrast, min font-size, semantic HTML, labels, buttons, what is a screen reader, what is aria, alt-text, and such
### Intermediate
"often used Aria" question (aria-label, aria-describedBy), visually hidden, screen reader only, role, and such
### Advanced
Advanced aria (aria-live, polite, expanded, and such), widgets

## Additional Information
:bulb: **Note:** I guess this concerns all the other topics as well

**Comment by jdwilkin4** [comment]
HI @YuriDevAT ! 

Thank you for your suggestion.

As you mentioned, this change of creating an option where users can select different levels would have to be a long term play and it would affect all categories.

So for now, we could open up an issue for contribution to ask for more beginner friendly accessibility questions to help first balance out the quiz questions.

But long term we would also need to also balance out the rest of the categories before we could implement this new feature.
Right now, some of the categories are balanced while others are weighted either heavily to the beginner side or the intermedia- advanced sides.

So we will need to start with fleshing out the quiz questions first for all categories before we could implement this feature so it works properly for all categories 👍

**Reply by YuriDevAT** [reply]
Thanks for taking the time reading through my suggestions. Means a lot to me how people at FreeCodeCamp trying to improve as best as possible 💟
